zlp-scheduler/app/views/layouts/application.html.haml

```diff
@@ -13,7 +13,7 @@
           // - else
           //   %h1{:style => "text-align:center"} Howdy!
       %header.header_div
-        = link_to image_tag("tamu_zlp_logo.png", class: "header"), "/"
+        = link_to image_tag("tamu_zlp_logo.png", class: "header", alt: "Tamu zlp logo"), "/"
```

Comment:
the html "alt" does not generated after upgrade rails to 5.2.4, so it is updated here